qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -89,6 +90,10 @@ class CommandFailed(CephTestException):
         "Stdout: %(stdout)s. Stderr: %(stderr)s.")
 
 
+class CephTestTimeout(Exception):
+    msg_fmt = "Operation timeout."
```

Comment:
What is this class attribute for?
```suggestion
    pass
```

Reply:
it's used by the base class as a default exception message: https://github.com/ceph/ceph/pull/49532/files/a2ba763ef0b61f7cd4c37bd178368da5a1c59e29#diff-4dd84999055f6b4243c43d045cf2f36a1cecbc3c5ec6cb2131c0f3aa80d3e648R81